src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,24 +14,399 @@ tree_item_index: 1100
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image20
+    name: aspera-image20.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image20](nx_asset://0a9c8974-1796-47c6-99b9-68aff556583e ?border=true)
+
+{{#> callout type='note'}}
+
+“Delete” does remove the content from the Aspera Node in most cases, however, we have seen some inconsistencies.  It is advised that you **regularly clean** your Aspera Transfer Node. Deleting the transfer does not remove the content from the repository if the transfer has already been completed. This ONLY deletes the transfer “bucket” or “wrapper”)
+{{/callout}}
+
+
+
+ 
+#### Transfers
+Create a transfer
+To create a transfer from the dashboard, simply click **+ New Upload** in the upper right hand corner.  By default, this will set the “target document” (or destination) as the user’s personal workspace.  This can be edited using the “common metadata” edit.
+
+
+On this screen, different actions are available; 
+
+You can:
+- Drag and drop (or click to select) file(s) or folder(s)  to upload with Aspera and follow the status of the uploads (whether you have the Aspera desktop client or not).
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image1
+    name: aspera-image1.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image1](nx_asset://43f35a20-386f-4600-a94f-aa7749e1df2a ?border=true)
+ 
+- Define/edit the common metadata, including the “target document” or the destination of the current transfer (set of uploaded files) by clicking on the Edit button. 
+{{#> callout type='info'}}
+By default, when clicking "+ New Upload" from the Aspera Dashboard or the transfer view, the `target location` for the upload is the **user's personal workspace**.  If you use the "Upload with Aspera" action from a folderish document, the `target location` will be set to **that folderish document**, by default.
+{{/callout}}
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image7
+    name: aspera-image7.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image7](nx_asset://6f83fbc6-7402-4552-9bd7-54919ff9db0b ?border=true)
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Aspera Connector/aspera-image15
+    name: aspera-image15.png
+    addins#screenshot#up_to_date
+--}}
+![aspera-image15](nx_asset://97069cfe-16c9-4d54-bd8f-a578d88aa6aa ?border=true)
+
+ 
+ 
+- Modify the permissions to share the transfer with other user(s) (i.e. another user is responsible for modifying the metadata).
```

Comment:
Guess `e.g.` may make more sense, since it just one possible use case 🤷🏼‍♂️ 
```suggestion
- Modify the permissions to share the transfer with other user(s) (e.g. another user is responsible for modifying the metadata).
```